server/libs/logger/logger-express.ts

```diff
@@ -0,0 +1,11 @@
+import { stdTimeFunctions } from 'pino'
+import { pinoHttp } from 'pino-http'
+
+const isProd = process.env.NODE_ENV === 'production'
```

Comment:
do we know if this is working as intended? i believe there was an issue with this env variable for the cleanAndPopulate?

Reply:
Yes! We are going to need to resolve #975 but this is the logic that we are going to need to follow. I will try to solve this today.

Reply:
Opened PR #984